buildSrc/src/main/kotlin/jacoco-kotlin-jvm.gradle.kts

```diff
@@ -25,6 +25,11 @@
  */
 
 import io.spine.internal.gradle.buildDirectory
+import java.io.File
```

Comment:
The changes in this file are a mystery to me. How all of that worked previously without imports, and why the warning suppression is no longer needed, I have no idea.